src/hooks/viewStateFacet/useGetSarcophagusArchaeologist.ts

```diff
@@ -0,0 +1,19 @@
+import { useContractRead } from 'wagmi';
+import { ViewStateFacetABI } from '../../abis/ViewStateFacet';
+
+export function useGetSarcophagusArchaeologist({
+  sarcoId,
+  archaeologist,
+}: {
+  sarcoId: string | number;
```

Comment:
why is this one a number also?

Reply:
I did that because we were planning to change the sarcoId to a number and I didn't want there to be a conflict.